tools/run_tests/run_tests.py

```diff
@@ -893,6 +899,37 @@ def runs_per_test_type(arg_str):
         msg = '\'{}\' is not a positive integer or \'inf\''.format(arg_str)
         raise argparse.ArgumentTypeError(msg)
 
+
+def _python_config_generator(name, major, minor, bits, config_args):
```

Comment:
For code elements introduced in this change, place them earlier in the file ("above") any of their use-sites.